skipper.go

```diff
@@ -325,14 +325,14 @@ type Options struct {
 	// DisabledFilters is a list of filters unavailable for use
 	DisabledFilters []string
 
-	// CloneRoute is a PreProcessor, that will be applied to all routes automatically. It
-	// will clone all matching routes and apply changes to the
+	// CloneRoute is an array PreProcessor's, that will be applied to all routes
+	// automatically. They will clone all matching routes and apply changes to the
 	// cloned routes.
-	CloneRoute *eskip.Clone
+	CloneRoute *[]eskip.Clone
```

Comment:
I guess it should be `[]*eskip.Clone`

Reply:
I do agree that `*[]eskip.Clone` is the worst choice I could make. :-)
However, due to Go slices internals ( https://go.dev/blog/slices-intro ) it seems to me I would try to make it `[]eskip.Clone` because it would decrease number of references levels.

Reply:
Okay, I tried to make it `[]eskip.Clone` but due to `eskip.NewClone` returning `*Clone` it seems it won't help to decrease reference levels. Will keep it `[]*eskip.Clone`.